style.css

```diff
@@ -238,22 +228,26 @@ body {
   cursor: pointer;
   position: relative;
   bottom: 114px;
-  height: 53px;
+  height: 49px;
   right: 28%;
 }
 
 .about-section {
   position: relative;
   left: 0.2px;
-  outline: 1.3px solid #e5e5e5;
-  outline-offset: 19px;
   border-top-right-radius: 98px;
+  margin-top: 90px;
+  border-top: 1px solid #e5e5e5;
```

Comment:
Well done on your efforts! 👏 It seems there might be a misunderstanding regarding the previous reviewer's request. They actually asked you to remove this border in the about us section, not the background in the page. Please keep the gray background as it is was but remove the gray border. By making this adjustment, you will align the element with the design requirements. Keep up the good work! 💪

Reply:
Thank you very much. I am not understand where i place gray border. Can you help